funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_manager.py

```diff
@@ -566,8 +566,8 @@ def cli_run():
     try:
         global logger
         # TODO Update logger to use the RotatingFileHandler in the funcx.utils.loggers.set_file_logger
-        logger = set_file_logger('{}/{}/manager.log'.format(args.logdir, args.uid),
-                                 name='funcx_manager',
+        logger = set_file_logger(os.path.join(args.logdir, args.uid, 'manager.log'),
+                                 name=__name__,
```

Comment:
Changing the log name will change the current log routing/aggregation behavior. Please remove this, and similar changes below.